ServicesDelete.php

```
@@ -0,0 +1,237 @@
+                $confirmed = confirm('Are you sure you want to delete this database?');
+                if (! $confirmed) {
+                    continue;
+                }
+                if (!$this->promptAndValidatePassword()) {
+                    continue;
+                }
+                DeleteResourceJob::dispatch($toDelete);
+                $this->info('Database deletion dispatched.');
+            }
+        }
+    }
```

PHPMD (2.15.0)

The method deleteDatabase() has a Cyclomatic Complexity of 14. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)